src/settings/target/mod.rs

```diff
@@ -33,6 +33,22 @@ impl Site {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct TemplateConfig {
+    #[serde(rename = "type")]
+    pub target_type: TargetType,
+    pub webpack_config: Option<String>,
```

Comment:
perhaps we can explicitly specify that this is a path? I think this would be helpful and provide more clarity.

Reply:
Out of scope for this - the `webpack_config` type on `Manifest` is an `Option<String>` so that's what it is here

Reply:
gotcha!